pkg/api/client/client.go

```diff
@@ -67,6 +67,12 @@ type client struct {
 	logger *logrus.Logger
 }
 
+func buildURL(c *client, uri string) string {
+	u, _ := url.Parse(fmt.Sprintf("%s://%s:%d%s", c.scheme, c.host, c.port, uri))
+
+	return u.String()
+}
+
```

Comment:
Is this change intended? It seems as same code just moved to another place. I'd prefer it as another commit or PR.